### DISCUSSION TITLE
Drag and Drop for buttons configurator

### DISCUSSION THREAD

**Discussion by JayPiEs** [bitfocus/companion#3093] [Ideas]
The configuration and adjustment of buttons is a thing that annoys me since version 2 of companion.
technically it is no problem anymore to provide drag and drop functionality in browsers and it would be a great advancement if the buttons configurator can be handled with drag and drop.

**Comment by dnmeid** [comment]
Yes, that would be really good. Do you want to program it and do a pull request?

**Reply by JayPiEs** [reply]
unfortanetely i am not a coder, i have the hope, that someone reads this and has the time to implement that feature to companion... ;-)

i found some information about drag and drop in html/browser:

https://www.w3schools.com/html/html5_draganddrop.asp

https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API

**Reply by dnmeid** [reply]
Thanks, we are well aware of drag and drop technology. Applying presets works with drag and drop.
